### PR TITLE
change copy/paste styles shortcuts

### PR DESCRIPTION
**Pull request by dwelle**
fix https://github.com/excalidraw/excalidraw/issues/1688

Moved to `Cmd/Ctrl+Opt/Alt+C/V` for copy/paste styles, because previous shortcuts collided with browser default.

overrides Mac's "inspect element", but that shortcut is only an alias for cross-platform `Cmd+Shift+C`, and Figma uses this also, so it should be fine.

### CODE REVIEW

File: src/keys.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export const KEYS = {
   Z_KEY_CODE: 90,
   GRID_KEY_CODE: 222,
```
Comment (lipis) [inline src/keys.ts:19]: Not part of this PR.. but can we change this to `APOSTROPHE_KEY_CODE`?

**Comment by Fausto95** [comment]
Works on Azerty ✅
